src/main/java/de/adesso/kicker/user/trackedranking/persistence/TrackedRanking.java

```diff
@@ -19,18 +19,22 @@
     private String rankingId;
 
     private int rank;
-
     private int rating;
 
+    private long wins;
```

Comment:
Wins and losses on `TrackedRanking` has already been added in the "statistics" branch.